coalib/io/FileFactory.py

```diff
@@ -0,0 +1,127 @@
+import os
+
+from coala_utils.decorators import generate_eq
+
+
+@generate_eq('name', 'timestamp')
+class FileFactory:
+    """
+    The ``FileFactory`` is a wrapper class to replace the actual file contents
+    from the ``file-dict``. The object contains the following information about
+    the file:
```

Comment:
Not sure we should mention the file-dict part here. I would rather keep it inside the cEPs, so once file-dicts are history, nobody will ever wonder what this means.